src/pages/students.js

```diff
@@ -0,0 +1,346 @@
+                </Surface>
+              </div>
+            </div>
+          </section>
+          <section className={styles.offerSection}>
+            <h2>What does the Student Edition offer?</h2>
+            <p>
+              There's no substitute for hands-on experience. That’s why we
+              designed the New Relic Student Edition,
+              <br /> which gives students and educators full access to New Relic
+              One, including:
+            </p>
+          </section>
+          <section className={cx(styles.section, styles.championProgram)}>
+            <div className={styles.point}>
+              <FeatherIcon
+                className={styles.pointIcon}
+                name="users"
+                size="4rem"
+              />
+              <h4>Three Full Users</h4>
+              <p>
+                Work with your friends and classmates like a real-world
+                production team
+              </p>
+            </div>
+            <div className={styles.point}>
+              <FeatherIcon
+                className={styles.pointIcon}
+                name="database"
+                size="4rem"
+              />
+              <h4>500 GB/month of Data Ingest</h4>
+              <p>
+                Collect, analyze, and alert on all your metrics, events, logs,
+                and traces from any source
+              </p>
+            </div>
+            <div className={styles.point}>
+              <FeatherIcon
+                className={styles.pointIcon}
+                name="cpu"
+                size="4rem"
+              />
+              <h4>Free Tier AI</h4>
+              <p>
+                Instantly detect, diagnose, and resolve issues before they
+                become a problem
+              </p>
+            </div>
+            <Button
+              as={ExternalLink}
+              className={styles.nominateButton}
+              href="https://forms.gle/Zkdub5e1x4MNqSKW9"
+              variant={Button.VARIANT.PRIMARY}
+            >
+              Sign up for a free account{' '}
+            </Button>
+          </section>
+          <section className={styles.faqSection}>
+            <h2>Frequently Asked Questions</h2>
+            <div className={styles.group}>
+              <p className={styles.question}>
+                How do I access New Relic Student Edition?
+              </p>
+              <div className={styles.answer}>
+                <p>
+                  New Relic offers the Student Edition exclusively through the
+                  GitHub Student Pack. To verify your account's eligibility,
+                  please authorize your Github account.
+                </p>
+                <p>
+                  Learn more about GitHub Education if you don't have GitHub
+                  Student Developer Pack yet.
+                </p>
+                <ul>
+                  <li>
+                    Sign up up for Student Edition - you will be prompted to
```

Comment:
Typo:

> Sign up for...

Split into two sentences:

> Student Edition. You'll be prompted to...

Reply:
Also, the other list items end with a period. This should be consistent. I recommend removing the periods, because they're all single-sentence items (our style guides eliminate the period). But if you keep the periods, you should add it here too.